src/Symfony/Component/Finder/Iterator/RecursiveDirectoryIterator.php

```diff
@@ -64,7 +64,7 @@ public function current()
         // the logic here avoids redoing the same work in all iterations
 
         if (null === $subPathname = $this->subPath) {
-            $subPathname = $this->subPath = (string) $this->getSubPath();
+            $subPathname = $this->subPath = $this->getSubPath();
```

Comment:
@nicolas-grekas You add this cast in https://github.com/symfony/symfony/commit/f156de6b71da006bd12642d2f7cec9668d5eb61d#diff-7fed51fcfebacfe775c561ca3f4704a2db761e2ed9c94890be7dd056c47e74dcR72, do you know why?